src/components/List/List.tsx

```diff
@@ -90,7 +108,13 @@ class List extends UIComponent<Extendable<IListProps>, any> {
     const { items } = this.props
     const itemProps = _.pick(this.props, List.itemProps)
 
-    return _.map(items, item => ListItem.create(item, { defaultProps: itemProps }))
+    return _.map(items, (item, idx) => {
+      itemProps.focusableItemProps = this.focusContainer.assignAtomicItemsProps(idx, items.length)
```

Comment:
probably, `prepareItemProps` or `createItemProps` (with `Item` being a singular form) would be better, due to this method
- prepares props for single item element
- there is no assignment for the result props object made there
- probably, we don't need to repeat `Atomic` in method's names